de-lib/src/main/java/org/iplantc/de/fileViewers/client/views/ReactVideoViewer.java

```diff
@@ -0,0 +1,24 @@
+package org.iplantc.de.fileViewers.client.views;
+
+import gwt.react.client.components.ReactClass;
+import gwt.react.client.proptypes.BaseProps;
+import jsinterop.annotations.JsPackage;
+import jsinterop.annotations.JsProperty;
+import jsinterop.annotations.JsType;
+
+/**
+ * @author aramsey
+ */
+
+@JsType(isNative = true, namespace = "CyVerseReactComponents.fileViewers", name = "VideoViewer")
```

Comment:
If `ReactVideoViewer` had the annotation `@JsType(isNative = true, namespace = "CyVerseReactComponents", name = "fileViewers")`, then there wouldn't be a need to specify `namespace` or `name` for `VideoViewer` below, since `gwt-react` would translate `ReactVideoViewer` to `fileViewers` under the `CyVerseReactComponents` namespace = `CyVerseReactComponents.fileViewers`.

So `ReactVideoViewer.VideoViewer` would translate to `CyVerseReactComponents.fileViewers.VideoViewer`, which is what `@JsProperty(namespace = "CyVerseReactComponents.fileViewers", name = "VideoViewer")` is also specifying.

For example: https://github.com/cyverse-de/ui/blob/493650b8e8daeb69ecf28296585ad636b33abe74/de-lib/src/main/java/org/iplantc/de/admin/desktop/client/metadata/view/ReactMetadataAdminViews.java#L11-L15